components/raftstore-v2/src/raft/peer.rs

```diff
@@ -36,6 +36,7 @@ const REGION_READ_PROGRESS_CAP: usize = 128;
 pub struct Peer<EK: KvEngine, ER: RaftEngine> {
     raft_group: RawNode<Storage<EK, ER>>,
     tablet: CachedTablet<EK>,
+    tombstone_tablet: Option<EK>,
```

Comment:
A peer may be split multiple times, this is not safe.

Reply:
Let's say tablet1 -> tablet2 -> tablet3, tablet3 can only be split after tablet2 is flushed, which means tablet1 can be deleted?

Reply:
But not all other split peers have finished initialization. For example, peer a is split into (b, a) and then into (b, c, a). b might be still in the progress of initialization.

Reply:
Could you be more verbose on the exact constraint here? I don't see how it relates to multiple times of splitting. My understanding is a's old tablet can be deleted after (1) a's persisted apply is advanced (2) split peers' tablet (b in your example) has been initialized (i.e. renamed from temporary path).